test/gui/shared/steps/steps.py

```diff
@@ -517,6 +517,16 @@ def step(context):
     )
     activateItem(waitForObjectItem(names.settings_QMenu, "Force sync now"))
 
+    # It is not guranteed that the object showing warning message will appear during the force sync.
+    # If the message appears, we need to click on the "Remove all files" button. Else, we should carry on with rest of the steps in test without any action.
+    # It takes upto 2000 ms for the object to appear. So, I put the timeout to be 2000 ms.
```

Comment:
no you haven't ;) I see a timeout of 1000ms

Reply:
I changed it for try and forgot to revert it back 🤣 Thanks for noticing.

Reply:
Delete tests need some adjustments alongside using the latest middleware. I suspect currently we are getting the false-positive results from delete test scenarios.

Reply:
I have fixed it and squashed the commits. Can you review it again?